gnomad/sample_qc/relatedness.py

```diff
@@ -1298,3 +1301,442 @@ def _get_alt_count(locus, gt, is_female):
+    However, neither approach explicitly documented how to compute *de novo*
+    probabilities for hemizygous genotypes in XY individuals. To address this,
+    we provide the full set of equations in this docstring.
+
+    The posterior probability of an event being truly *de novo* vs. the probability it was a missed heterozygote call in one of the two parents is:
+
+    .. math::
+
+        P_{dn} = \frac{P(DN \mid \text{data})}{P(DN \mid \text{data}) + P(\text{missed het in parent(s)} \mid \text{data})}
+
+    The terms are defined as follows:
+
+    - :math:`P(DN \mid \text{data})` is the probability that the variant is *de novo*, given the observed genotype data.
+
+    - :math:`P(\text{missed het in parent(s)} \mid \text{data})` is the probability that the heterozygous variant was **missed in at least one parent**.
+
+    Applying Bayesian Theorem to the numerator and denominator yields:
+
+    .. math::
+
+        P_{dn} = \frac{P(\text{data} \mid DN) \cdot P(DN)}{P(\text{data} \mid DN) \cdot P(DN) + P(\text{data} \mid \text{missed het in parent(s)}) \cdot P(\text{missed het in parent(s)})}
+
+    where:
+
+    - :math:`P(\text{data} \mid DN)`: Probability of observing the data under the assumption of a *de novo* mutation.
+
+      - **Autosomes and PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in father}) \cdot P(\text{hom_ref in mother}) \cdot P(\text{het in proband})
+
+      Probability of a observing a *de novo* mutation given the data specifically for hemizygous calls in XY individuals
+
+          Note that hemizygous calls in XY individuals will be reported as homozygous alternate without any sex ploidy adjustments, which is why the formulas below use `P(hom_alt in proband)`
+
+      - **X non-PAR regions (XY only)**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in mother}) \cdot P(\text{hom_alt in proband})
+
+      - **Y non-PAR regions (XY only)**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in father}) \cdot P(\text{hom_alt in proband})
+
+    - :math:`P(DN)`: The prior probability of a *de novo* mutation from literature, defined as:
+
+      .. math::
+
+          P(DN) = \frac{1}{3 \times 10^7}
+
+    - :math:`P(\text{data} \mid \text{missed het in parent(s)})`: Probability of observing the data under the assumption of a missed het in a parent.
+
+      - **Autosomes and PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid \text{missed het in parents}) = ( P(\text{het in father}) \cdot P(\text{hom_ref in mother}) + P(\text{hom_ref in father}) \cdot P(\text{het in mother})) \cdot P(\text{het in proband})
+
+      - **X non-PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid \text{missed het in mother}) = (P(\text{het in mother}) + P(\text{hom_alt in mother})) \cdot P(\text{hom_alt in proband})
+
+      - **Y non-PAR regions**:
```

Comment:
these are also for XY samples only

Reply:
can you add a note here reflecting this for consistency with the documentation above?